packages/ui/src/views/repo/pull-request/details/components/conversation/pull-request-comment-box.tsx

```diff
@@ -71,72 +69,115 @@ const PullRequestCommentBox = ({
   const handleDropForUpload = async (event: any) => {
     handleFileDrop(event, handleUploadCallback)
   }
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  const handlePasteForUpload = (event: { preventDefault: () => void; clipboardData: any }) => {
+
+  const handlePasteForUpload = (event: React.ClipboardEvent) => {
     handlePaste(event, handleUploadCallback)
   }
 
+  // TODO: add the remaining required logic for the toolbar
+  const toolbar: ToolbarItem[] = useMemo(() => {
+    const initial: ToolbarItem[] = []
+    return [
+      ...initial,
+      { icon: 'suggestion', action: ToolbarAction.SUGGESTION },
+      { icon: 'header', action: ToolbarAction.HEADER },
+      { icon: 'bold', action: ToolbarAction.BOLD },
+      { icon: 'italicize', action: ToolbarAction.ITALIC },
+      { icon: 'attachment', action: ToolbarAction.UPLOAD, onClick: handleFileSelect },
+      { icon: 'list', action: ToolbarAction.UNORDER_LIST },
+      { icon: 'checklist', action: ToolbarAction.CHECK_LIST },
+      { icon: 'code', action: ToolbarAction.CODE_BLOCK }
+    ]
+  }, [])
+
   return (
-    <div className="flex items-start space-x-4">
-      {(!isEditMode || !hideAvatar) && avatar}
+    <div className="flex items-start gap-x-3">
+      {!inReplyMode && avatar}
+
       <div
-        className={cn('min-w-0 flex-1 px-4 pb-5 pt-1.5', {
+        className={cn('pb-5 pt-1.5 px-4 flex-1 bg-background-2 border-border-1', {
           'border rounded-md': !inReplyMode || isEditMode,
-          'border-t ': inReplyMode
+          'border-t': inReplyMode
         })}
       >
         <Tabs variant="tabnav" defaultValue="write">
           <TabsList className="relative left-1/2 w-[calc(100%+32px)] -translate-x-1/2 px-4">
-            <TabsTrigger value="write">Write</TabsTrigger>
-            <TabsTrigger value="preview">Preview</TabsTrigger>
+            <TabsTrigger className="data-[state=active]:bg-background-2" value="write">
+              Write
+            </TabsTrigger>
+            <TabsTrigger className="data-[state=active]:bg-background-2" value="preview">
+              Preview
+            </TabsTrigger>
           </TabsList>
 
           <TabsContent className="mt-4" value="write">
             <div
-              onDrop={e => {
-                handleDropForUpload(e)
-              }}
-              onPaste={handlePasteForUpload}
-              className="relative gap-y-1"
-              onDragOver={event => {
-                event.preventDefault()
-              }}
+              className="relative"
+              onDrop={handleDrop}
+              onDragOver={e => e.preventDefault()}
+              onDragEnter={handleDragEnter}
+              onDragLeave={handleDragLeave}
+              ref={dropZoneRef}
             >
               <Textarea
-                onDrop={e => {
-                  handleDropForUpload(e)
-                }}
+                className="min-h-24 p-3 pb-10"
```

Comment:
![image](https://github.com/user-attachments/assets/180aa14d-dc15-4c2f-b694-50d0961f0c25)
seems to get overlay buttons if text is past height